app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/componentHealthCheckControls/componentHealthCheckControls.jsx

```diff
@@ -0,0 +1,145 @@
+    defaultMessage: 'Component',
+  },
+  passingRateValidationError: {
+    id: 'CumulativeTrendControls.PassingRateValidationError',
+    defaultMessage: 'Should have value from 50 to 100',
+  },
+  attributeKeyValidationError: {
+    id: 'CumulativeTrendControls.attributeKeyValidationError',
+    defaultMessage: 'Value should have size from 1 to 128',
+  },
+  attributesArrayValidationError: {
+    id: 'CumulativeTrendControls.attributesArrayValidationError',
+    defaultMessage:
+      'Enter an attribute key whose unique value will be used for combine tests into groups',
+  },
+});
+const validators = {
+  passingRate: (formatMessage) => (value) =>
+    (!value || !validate.inRangeValidate(value, 50, 100)) &&
+    formatMessage(messages.passingRateValidationError),
+  attributeKey: (formatMessage) => (value) => {
+    if (!value) {
+      return formatMessage(messages.attributesArrayValidationError);
+    } else if (!validate.attributeKey(value)) {
+      return formatMessage(messages.attributeKeyValidationError);
+    }
+    return undefined;
+  },
+};
+
+@connect((state) => ({
+  itemAttributeKeysAllSearch: URLS.itemAttributeKeysAllSearch(activeProjectSelector(state)),
+}))
+@injectIntl
+export class ComponentHealthCheckControls extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    widgetSettings: PropTypes.object.isRequired,
+    initializeControlsForm: PropTypes.func.isRequired,
+    formAppearance: PropTypes.object.isRequired,
+    onFormAppearanceChange: PropTypes.func.isRequired,
+    itemAttributeKeysAllSearch: PropTypes.string.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    const { widgetSettings, initializeControlsForm } = props;
+
+    initializeControlsForm({
+      contentParameters: widgetSettings.contentParameters || {
+        itemsCount: DEFAULT_ITEMS_COUNT,
+        contentFields: [],
+        widgetOptions: {},
+      },
+    });
+  }
+
+  normalizeValue = (value) => value && `${value}`.replace(/\D+/g, '');
+
+  formatFilterValue = (value) => value && value[0];
+  parseFilterValue = (value) => value && [value];
+
+  renderAttributesFieldArray = ({ fields, fieldValidator }) => (
+    <AttributesFieldArrayControl
+      fields={fields}
+      fieldValidator={fieldValidator}
+      maxAttributesAmount={MAX_ATTRIBUTES_AMOUNT}
+      url={this.props.itemAttributeKeysAllSearch}
+    />
+  );
+
+  render() {
+    const {
+      intl: { formatMessage },
+      formAppearance,
+      onFormAppearanceChange,
+    } = this.props;
+
+    return (
+      <Fragment>
+        <FieldProvider name="filters" parse={this.parseFilterValue} format={this.formatFilterValue}>
+          <FiltersControl
+            formAppearance={formAppearance}
+            onFormAppearanceChange={onFormAppearanceChange}
+          />
+        </FieldProvider>
+
+        {!formAppearance.isMainControlsLocked && (
+          <div className={cx('component-health-check-wrap')}>
+            <ScrollWrapper>
```

Comment:
Just pass next props to the `ScrollWrapper`:
`hideTracksWhenNotNeeded`, `autoHeight`, `autoHeightMax={max height here}`.
Now you can remove `component-health-check-wrap` CSS class and `div` wrapper